build-logic/src/main/kotlin/com/adevinta/spark/SparkPublishingPlugin.kt

```diff
@@ -54,17 +57,14 @@ internal class SparkPublishingPlugin : Plugin<Project> {
                 name = "Local"
                 url = uri(rootProject.layout.buildDirectory.dir(".m2/repository"))
             }
-            maven {
-                name = "OSSRH"
-                url = when (version.toString().endsWith("-SNAPSHOT")) {
-                    true -> "https://s01.oss.sonatype.org/content/repositories/snapshots/"
-                    false -> "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
-                }.let(::uri)
-                credentials {
-                    username = System.getenv("OSSRH_USERNAME")
-                    password = System.getenv("OSSRH_TOKEN")
-                }
-            }
+        }
+    }
+
+    private fun Project.configureNMCP() = configure<NmcpExtension> {
+        centralPortal {
+            username = System.getenv("CENTRAL_PORTAL_USERNAME")
+            password = System.getenv("CENTRAL_PORTAL_PASSWORD")
+            publishingType = "USER_MANAGED"
```

Comment:
We could migrate to the automated publishing to simplify this a bit more now that we validated the new plugin.